internal/reflectx/reflectx_test.go

```diff
@@ -0,0 +1,70 @@
+package reflectx
+
+import (
+	"testing"
+
+	"github.com/ooni/probe-cli/v3/internal/testingx"
+)
+
+type example struct {
+	_    struct{}
+	Age  int64
+	Ch   chan int64
+	F    bool
+	Fmp  map[string]*bool
+	Fp   *bool
+	Fv   []bool
+	Fvp  []*bool
+	Name string
+	Ptr  *int64
+	V    []int64
```

Comment:
```suggestion
	V    []int64
	p    string          
```
Although `_` is a private field, I think we should another private field explicitly to clarify that we only check for public fields using the `IsExported` check.

Reply:
✔️